### PR TITLE
add JiriJAdam's hackbar

### PR DESCRIPTION
**Pull request by JiriJAdam**
I added the planet over profit image to give people the idea about what the government/corporations really do care about the most. As well as gave a statement about how open source will change the world and future to the better. To give out voices to everyone who have great ideas for projects/products.

### CODE REVIEW

File: _hackbars/jirijadam.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+author: Jiri Adam Jr
+---
+
+We should all focus on open source for the better future!
+
+---
+
+## We need to lose profits and focus on time and our environment!
+
+![image_caption]({{site.baseurl}}/assets/images/jirijadam.jpg)
```
Comment (tasnimzotder) [inline _hackbars/jirijadam.md]:
```suggestion
![image_caption]({{site.baseurl}}/assets/images/jirijadam.jpeg)
```

**Comment by JiriJAdam** [comment]
Did the changes work?

**Review by tasnimzotder — APPROVED** [review]

**Comment by tasnimzotder** [comment]
> Did the changes work?

Yes. It works.

**Comment by tasnimzotder** [comment]
Thanks for your contributions :)

**Pull request merged**